File: app/src/main/java/com/bubelov/coins/api/external/BitstampApi.java
package com.bubelov.coins.api.external;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Author: Igor Bubelov
 * Date: 9/28/15 10:03 PM
 */

public interface BitstampApi {
    @GET("ticker/")
    Call<BitstampTicker> getTicker();
}

File: app/src/main/java/com/bubelov/coins/api/rates/BitstampApi.kt
package com.bubelov.coins.api.rates

import retrofit2.Call
import retrofit2.http.GET

/**
 * @author Igor Bubelov
 */

interface BitstampApi {
    @GET("ticker/btcusd")
    fun getBtcUsdTicker(): Call<BitstampTicker>

    @GET("ticker/btceur")
    fun getBtcEurTicker(): Call<BitstampTicker>
}